README.md

```diff
@@ -171,6 +171,39 @@ ngrok http 8443 -host-header="localhost:8443"
 The frontend app is composed of a number of components that are documented in
 our [Storybook](https://wordpress.github.io/openverse-frontend).
 
+##### How to create a component
+
+To create a component you can run:
+
+```
+pnpm run create:component [component name in PascalCase]
+for example: pnpm run create:component VButtonGreen
+```
```

Comment:
```suggestion
```shell
# Make sure you have run `pnpm install` at least once before.
pnpm run create:component [component name in PascalCase]
for example: pnpm run create:component VButtonGreen
```
```